notebooks/Solution_Exercises.ipynb

```diff
@@ -0,0 +1,165 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "8c3882b9",
+   "metadata": {},
+   "source": [
+    "# Solutions of exercises\n",
+    "\n",
+    "Exercises are available in a few notebooks. Below are solutions to the exercises.\n",
+    "How to use the notebook:\n",
+    "\n",
+    "* **DO NOT RUN THE NOTEBOOK**\n",
+    "* **Copy the code snippet(s) and run the snippet(s) in the relevant notebook**\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "38dce9f9",
+   "metadata": {},
+   "source": [
+    "## Exercises from notebook idr0062_prediction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "75db4ae2",
+   "metadata": {},
+   "source": [
+    "### Exercise 1 "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "47d19053",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Question 1: Read the geojson file:\n",
+    "import json\n",
+    "with open(geojson_file) as f:\n",
+    "    d = json.load(f)\n",
+    "    s = json.dumps(d)\n",
+    "values = geojson.loads(s)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e8fa7860",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Question 2: Overlay the shapes from a specific plane on the image\n",
+    "z = 5\n",
+    "fiz, ax = plt.subplots()\n",
+    "plt.imshow(data[0, 0, z, :, :], cmap='gray')\n",
+    "for v in values:\n",
+    "    if v.properties[\"z\"] == z:\n",
+    "        label_geojson = numpy.array(v.geometry.coordinates)\n",
+    "        p = label_geojson.squeeze()\n",
+    "        ax.plot(p[:, 0], p[:, 1], linewidth=v.properties[\"stroke-width\"], color=\"red\")\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dddaad7f",
+   "metadata": {},
+   "source": [
+    "### Exercise 2 \n",
+    "\n",
+    "In order to save the StarDist labels to an OMERO.server with write access, you will first need to log in and change the image ID."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "e0d0c98b",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "False\n"
+     ]
+    }
+   ],
+   "source": [
+    "from omero.gateway import BlitzGateway\n",
+    "HOST = 'YOUR_SERVER'\n",
+    "conn = BlitzGateway('YOUR_USERNAME', 'YOUR_PASSWORD',\n",
```

Comment:
could we have this in a popup which would hide the pwd as I am typing it in ? Otherwise I cannot show it in the video